ajax/libs/jquery.loadTemplate/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "jquery.loadTemplate",
+    "description": "A template plugin for jQuery. Allows templating without cluttering JavaScript code with markup",
+    "filename": "jquery.loadTemplate.min.js",
+    "keywords": [
+       "templates",
+       "templating",
+       "jquery-plugin"
+    ],
+    "author": "codepb",
+    "homepage": "http://codepb.github.io/jquery-template/",
```

Comment:
Let's use `https` because it's more security for browsing. to prevent the possibility the users been attacked during the redirection.
`https://codepb.github.io/jquery-template/`
Thank you.